### PR TITLE
chore: update invoice summaries by user

### PR DESCRIPTION
**Pull request by Onitsiky**

### CODE REVIEW

File: src/main/java/app/bpartners/api/service/event/RefreshInvoiceSummaryTriggeredService.java [hunk 1]
```diff
@@ -10,10 +14,22 @@
...
 public class RefreshInvoiceSummaryTriggeredService
     implements Consumer<RefreshInvoiceSummaryTriggered> {
-  private final InvoiceSummaryService invoiceSummaryService;
+  private final UserService userService;
+  private final EventProducer eventProducer;
 
   @Override
   public void accept(RefreshInvoiceSummaryTriggered refreshInvoiceSummaryTriggered) {
-    invoiceSummaryService.updateInvoicesSummary();
+    getEnabledUsersIds();
```
Comment (hei-ryan) [inline src/main/java/app/bpartners/api/service/event/RefreshInvoiceSummaryTriggeredService.java]: Rename this method. It actually says "get enabled users ids" but what it does is "sending InvoiceSummaryRefresh event" ...

File: src/test/java/app/bpartners/api/integration/InvoiceSummaryServiceIT.java [hunk 2]
```diff
@@ -24,12 +24,14 @@ public class InvoiceSummaryServiceIT extends MockedThirdParties {
 
   @Test
   void update_invoices_summaries_ok() {
-    List<User> enabledUsers =
+    List<String> enabledUsers =
```
Comment (hei-ryan) [inline src/test/java/app/bpartners/api/integration/InvoiceSummaryServiceIT.java]: enabledUsersIds

File: src/main/java/app/bpartners/api/endpoint/event/gen/RefreshInvoiceSummaryByUserTriggered.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import javax.annotation.processing.Generated;
+import lombok.*;
+
+@Generated("EventBridge")
+@EqualsAndHashCode
+@Builder
+@ToString
+@AllArgsConstructor
+@Data
+public class RefreshInvoiceSummaryByUserTriggered implements Serializable {
```
Comment (hei-ryan) [inline src/main/java/app/bpartners/api/endpoint/event/gen/RefreshInvoiceSummaryByUserTriggered.java]: Why not just RefreshUserInvoiceSummaryTriggered ?

**Review by hei-ryan — CHANGES_REQUESTED** [review]

File: src/main/java/app/bpartners/api/endpoint/event/gen/RefreshInvoiceSummaryByUserTriggered.java (hunk 3 above)
Comment (Onitsiky) [reply src/main/java/app/bpartners/api/endpoint/event/gen/RefreshInvoiceSummaryByUserTriggered.java]: I already have the RefreshUserInvoiceSummaryTriggered, which is the one which is invoked in the scheduled task in EventBridge

**Pull request merged**